waiter/src/waiter/settings.clj

```diff
@@ -420,6 +420,11 @@
                                    :max-patch-retries 5
                                    :max-name-length 63
                                    :pod-base-port 31000
+                                   :pod-bypass-force-sigterm-secs 120
+                                   :pod-bypass-pre-stop-cmd ["/bin/sh"
+                                                             "-c"
+                                                             "echo $(date +%Y-%m-%dT%H:%M:%S.%3NZ) INFO preStop: sleeping for ${WAITER_BYPASS_FORCE_SIGTERM_SECS} secs >> /proc/1/fd/1 ; sleep ${WAITER_BYPASS_FORCE_SIGTERM_SECS} ; echo $(date +%Y-%m-%dT%H:%M:%S.%3NZ) INFO preStop: ending sleep, and container will receive SIGTERM shortly >> /proc/1/fd/1"]
```

Comment:
I did not make this a separate script. I think its short enough we can leave as a command here. In the future when we want to add support for short-circuiting when outstanding requests are zero, we can pull it out here.